### DISCUSSION TITLE
Lambda deployment issue

### DISCUSSION THREAD

**Discussion by jordimares** [bentoml/BentoML#1948] [Q&A]
Hi all and thanks in advance! 

I have a BentoService created which requires a pip package from our private repository. When I run `bentoml lambda deploy my-first-lambda-deployment -b UKValuationService:20211102094127_99694E` pip fails to find the private package and breaks the deployment. Is there any way I could add the extra url of our private repository so that pip is able to install the package during deploy?

Thanks!

**Comment by parano** [comment]
Hi @jordimares, have you tried using this https://github.com/bentoml/aws-lambda-deploy  - we've been migrating out the cloud deployment functionalities to independent projects and this branch has some improvements and bug fixes that are not available in the built-in CLI tool.
